pages/gallery.html

```diff
@@ -1,161 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="./../css/gallery.css" />
-    <link rel="stylesheet" href="./../css/style.css" />
-    <link rel="stylesheet" type="text/css" href="../css/home.css">
-    <link
-      href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap"
-      rel="stylesheet"
-    />
-    <title>Galeria - Hostal las paisas</title>
-  </head>
-  <body>
-    <section class="small-hero small-hero__image--gallery ">
-      <div class="small-hero__layer"></div>
-      <div class="small-hero__layer small-hero__layer--darker"></div>
-      <h1 class="small-hero__title">Galería</h1>
-    </section>
-    <nav id="main-nav" class="main-nav">
-      <div class="container main-nav__container">
-        <div id="main-nav__logo" class="main-nav__logo">
-          <img class="main-nav__logo-image" src="./../assets/images/logo-hdlp.svg" alt="Hostal Donde Las Paisas">
-        </div>
-        <div id="main-nav__burguer" class="main-nav__burguer">
-          <div class="main-nav__burguer-bar burguer__bar-1"></div>
-          <div class="main-nav__burguer-bar burguer__bar-2"></div>
-          <div class="main-nav__burguer-bar burguer__bar-3"></div>
-        </div>
-        <div id="main-nav__menu" class="main-nav__menu">
-          <ul id="main-nav__menu-items" class="main-nav__menu-items">
-            <li><a class="main-nav__menu-link" href="../index.html">Inicio</a></li>
-            <li><a class="main-nav__menu-link">Galería</a></li>
-            <li><a class="main-nav__menu-link" href="./quote.html">Cotizar</a></li>
-            <li><a class="main-nav__menu-link" href="./contact.html">Contáctanos</a></li>
-          </ul>
-        </div>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="./../css/gallery.css" />
+  <link rel="stylesheet" href="./../css/style.css" />
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet" />
+  <title>Galeria - Hostal las paisas</title>
+</head>
+
+<body>
+  <section class="small-hero small-hero__image--gallery ">
+    <div class="small-hero__layer"></div>
+    <div class="small-hero__layer small-hero__layer--darker"></div>
+    <h1 class="small-hero__title">Galería</h1>
+  </section>
+  <nav>
+    <!-- nav bar section -->
+  </nav>
```

Comment:
please don't remove the navbar and hero

Reply:
navbar section is not in the this gallery page.